packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -199,6 +201,17 @@ export const NotifiSubscriptionContextProvider: React.FC<
     }
   }, [frontendClient.userState]);
 
+  useEffect(() => {
+    frontendClient
+      .fetchData()
+      .then((data) => {
+        render(data);
+      })
+      .catch((_e) => {
+        throw new Error('error re-rendering data');
+      });
+  }, [reRenderTriggered]);
+
```

Comment:
hm, this feels like a code smell for a missing `render` somewhere -- I think instead of exposing an additional param, it would be better to plug the gap where we have the issue.

Reply:
Addressed. Exposing useCallback instead of useState + useEffect